bootstrap/viaproxy/src/main/java/org/geysermc/geyser/platform/viaproxy/GeyserViaProxyLogger.java

```diff
@@ -75,6 +75,13 @@ public void debug(String message) {
         }
     }
 
+    @Override
+    public void debug(String message, Object... parameters) {
+        if (this.debug) {
+            this.logger.debug(ConsoleFormatter.convert(message.formatted(parameters)));
```

Comment:
Bungee and Spigot loggers use `String.format(message, parameters)` and this uses `message.formatted(parameters)`, could we stick to one of them?